RecFCCeeCalorimeter/src/components/NoiseCaloCellsVsThetaFromFileTool.h

```diff
@@ -0,0 +1,127 @@
+#ifndef RECFCCEECALORIMETER_NOISECALOCELLSVSTHETAFROMFILETOOL_H
+#define RECFCCEECALORIMETER_NOISECALOCELLSVSTHETAFROMFILETOOL_H
+
+// from Gaudi
+#include "GaudiAlg/GaudiTool.h"
+#include "GaudiKernel/RndmGenerators.h"
+#include "GaudiKernel/IRndmGenSvc.h"
+
+// k4geo
+// #include "detectorSegmentations/FCCSWGridPhiEta_k4geo.h"
+
+// k4FWCore
+#include "k4Interface/INoiseConstTool.h"
+#include "k4Interface/INoiseCaloCellsTool.h"
+#include "k4Interface/ICellPositionsTool.h"
+
+class IGeoSvc;
+
+// Root
+class TH1F;
+
+/** @class NoiseCaloCellsVsThetaFromFileTool
+ *
+ *  Tool for calorimeter noise
+ *  Access noise constants from TH1F histogram (noise vs. theta)
+ *  createRandomCellNoise: Create random CaloHits (gaussian distribution) for the vector of cells
+ *  filterCellNoise: remove cells with energy below threshold*sigma from the vector of cells
```

Comment:
Won't we include this filtering in CreateCaloCells?

Reply:
the method is indeed called in CreateCaloCells: https://github.com/HEP-FCC/k4RecCalorimeter/blob/a902075091a928d40c3b6290792d1e2296401190/RecCalorimeter/src/components/CreateCaloCells.cpp#L108

What would you like to change?

Reply:
It can be done in another PR, but the filtering of cells could be isolated away from the noise tool. One may want to filter cells even without adding noise.